siege_utilities/geo/providers/nces_download.py

```
@@ -0,0 +1,388 @@
+            if lat_col is None or lon_col is None:
+                raise NCESDownloadError(
+                    f"Cannot find lat/lon columns in {csv_path}. "
+                    f"Available: {list(df.columns)}"
+                )
+
+            df[lat_col] = pd.to_numeric(df[lat_col], errors="coerce")
+            df[lon_col] = pd.to_numeric(df[lon_col], errors="coerce")
+            valid = df[lat_col].notna() & df[lon_col].notna()
+            df = df[valid]
+
+            geometry = [Point(lon, lat) for lon, lat in zip(df[lon_col], df[lat_col])]
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)